lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -194,6 +213,61 @@ private void clearComposition() {
     // Do nothing.
   }
 
+  /**
+   * Add a color filter to specific content on a specific layer.
+   * @param layerName name of the layer where the supplied content name lives
+   * @param contentName name of the specific content that the color filter is to be applied
+   * @param colorFilter the color filter, null to clear the color filter
+   */
+  public void addColorFilterToContent(String layerName, String contentName,
+      @Nullable ColorFilter colorFilter) {
+    addColorFilterInternal(layerName, contentName, colorFilter);
+  }
+
+  /**
+   * Add a color filter to a whole layer
+   * @param layerName name of the layer that the color filter is to be applied
+   * @param colorFilter the color filter, null to clear the color filter
+   */
+  public void addColorFilterToLayer(String layerName, @Nullable ColorFilter colorFilter) {
+    addColorFilterInternal(layerName, null, colorFilter);
+  }
+
+  /**
+   * Add a color filter to all layers
+   * @param colorFilter the color filter, null to clear all color filters
+   */
+  public void addColorFilter(ColorFilter colorFilter) {
+    addColorFilterInternal(null, null, colorFilter);
+  }
+
+  /**
+   * Clear all color filters on all layers and all content in the layers
+   */
+  public void clearColorFilters() {
+    colorFilterData.clear();
+    addColorFilterInternal(null, null, null);
+  }
+
+  /**
+   * Private method to capture all color filter additions.
+   * There are 3 different behaviors here.
+   * 1. layerName is null. All layers supporting color filters will apply the passed in color filter
+   * 2. layerName is not null, contentName is null. This will apply the passed in color filter
+   *    to the whole layer
+   * 3. layerName is not null, contentName is not null. This will apply the pass in color filter
+   *    to a specific composition content.
+   */
+  private void addColorFilterInternal(@Nullable String layerName, @Nullable String contentName,
+      @Nullable ColorFilter colorFilter) {
+    colorFilterData.add(new ColorFilterData(layerName, contentName, colorFilter));
```

Comment:
well it would make that entry have a null colorFilter because colorFilterData is a Set. But making the removal explicit will make it so we don't do extra work